README.md

```diff
@@ -41,6 +38,8 @@ You will need:
 
 - [for -UA configurations] USB A to USB Micro B cable
 
+We do not advice using a Raspberry Pi 3, in place of a Raspberry Pi 4. Lack of RAM on the Raspberry Pi 3 may cause the AVS device SDK compilation to fail, as reported in [this issue](https://github.com/alexa/avs-device-sdk/issues/2035).
```

Comment:
`advice` -> `advise`.  Advice is a noun while advise is a verb.

Reply:
Updated